internal/fnruntime/container.go

```diff
@@ -110,6 +115,25 @@ func (f *ContainerFn) Run(reader io.Reader, writer io.Writer) error {
 	return nil
 }
 
+// dockerCmdAvailable runs `docker version` to check that the docker command is
+// available, and returns an error with installation instructions if it is not.
+func dockerCmdAvailable() error {
+	const op errors.Op = "docker.check"
+
+	suggestedText := `Docker must be running to use this command.
+To install docker, follow the instructions at https://docs.docker.com/get-docker/
+`
+	buffer := &bytes.Buffer{}
+
+	cmd := exec.Command(dockerBin, "version")
+	cmd.Stderr = buffer
+	err := cmd.Run()
+	if err != nil {
+		return errors.E(op, fmt.Errorf("%s", suggestedText))
```

Comment:
don't use `errors.E` here, instead just return the `fmt.Errorf(...)`

Reply:
done